Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -57,6 +57,8 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
             return true
         case .loginMagicLinkEmphasis:
             return true
+        case .loginMagicLinkEmphasisM2:
+            return true
```

Comment:
❓ Do we really need the feature flag or will we apply A/B testing to this?

Reply:
I think it's worth keeping the feature flag because this is an experiment, and we might end up discarding this change based on the result. Next sprint we should be able to start A/B testing these changes.